cmd/nerdctl/login_linux_test.go

```diff
@@ -145,12 +148,15 @@ func TestLoginWithPlainHttp(t *testing.T) {
 			shouldUseInSecure: true,
 		},
 		{
-			regHost:           "127.0.0.1",
-			regPort:           80,
-			useRegPort:        false,
-			username:          "admin",
-			password:          "validTestPassword",
-			shouldSuccess:     false,
+			regHost:    "127.0.0.1",
+			regPort:    80,
+			useRegPort: false,
+			username:   "admin",
+			password:   "validTestPassword",
+			// Following the merging of the below, any localhost/loopback registries will
+			// get automatically downgraded to HTTP so this will still succceed:
+			// https://github.com/containerd/containerd/pull/7393
+			shouldSuccess:     true,
```

Comment:
@Zheaoli PTAL, is this expected?

Reply:
Unexpectedly, I think the PR7393 is not included in v1.6.8 which version is dependent by nerdctl

Reply:
In that case, I've opened a [backport request for the LogURIGenerator only]( https://github.com/containerd/containerd/pull/7632) and will revert this test and update go.mod once it gets in.
Thank you both!

Reply:
I'm not sure I get the point. `go.mod` in nerdctl should point to v1.7.0-beta.0 (or later), not v1.6.x.

Reply:
@aznashwan nerdctl depends on other [containerd commits](https://github.com/containerd/containerd/pull/6744) not present on `v1.6.x`; 
```
git tag -l --contains 3df7674058301f290fc148719d483e47f4118494 
v1.7.0-beta.0
```
We should keep pointing to `main` until `v1.17.x` I guess.

@AkihiroSuda what do you mean by expectation here ?

Reply:
> I'm not sure I get the point. go.mod in nerdctl should point to v1.7.0-beta.0 (or later), not v1.6.x.

My bad, I got confused by the version string currently in go.mod saying `1.6.1`.

It looks like [this is the containerd commit currently referenced in go.mod](https://github.com/containerd/containerd/pull/7257), which would put it on early August around the `v1.6.8` area.
 
Unfortunately the LogURIGenerator fixes we depend on are not included in `v1.6.9` either so I guess the only option is switching to ` v1.7.0-beta.0`, and keeping the changes to these local registry tests.

Please lemme know if `v1.7.0-beta.0` would be okay so I can update the PR and re-run the test suite a couple more times.